test/e2e/libvirt_test.go

```diff
@@ -14,6 +15,7 @@ func TestLibvirtCreateSimplePod(t *testing.T) {
 }
 
 func TestLibvirtCreatePodWithConfigMap(t *testing.T) {
+	t.Skip("Failing on CI")
```

Comment:
How about using an ENV value to skip it only when running in CI while allowing it otherwise?